src/shell/shell_cmds.cpp

```diff
@@ -844,6 +844,13 @@ void DOS_Shell::CMD_LS(char *args)
 
 	size_t w_count = 0;
 
+	constexpr int ansi_blue = 34;
+	constexpr int ansi_green = 32;
+	auto write_color = [&](int color, const std::string &txt, int width) {
+		const int padr = width - static_cast<int>(txt.size());
+		WriteOut("\033[%d;1m%s\033[0m%-*s", color, txt.c_str(), padr, "");
+	};
```

Comment:
Looks great!
Does it make sense to elevate the `write_color` function so it can be used anywhere `WriteOut` is currently used?

Reply:
`WriteOut` is a method in `Program` class, so it would need to be a another method. If there ever will be a need for it, then we could do it, but I guess almost all uses of coloured text will be for statically-known strings, where using escape codes directly is more flexible.

Reply:
> almost all uses of coloured text will be for statically-known strings

Ahh.. makes sense; yes - things would get messy having to breakup those static strings to insert colorizing variables.